tests/database/odbc-base/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Copyright Microsoft. 
```

Comment:
Please remove all the Microsoft copyright comments.

Reply:
done

Reply:
Copyright is still there.